client.go

```diff
@@ -124,14 +124,17 @@ func New(address string, opts ...ClientOpt) (*Client, error) {
 			grpc.WithConnectParams(connParams),
 			grpc.WithContextDialer(dialer.ContextDialer),
 			grpc.WithReturnConnectionError(),
-
-			// TODO(stevvooe): We may need to allow configuration of this on the client.
-			grpc.WithDefaultCallOptions(grpc.MaxCallRecvMsgSize(defaults.DefaultMaxRecvMsgSize)),
-			grpc.WithDefaultCallOptions(grpc.MaxCallSendMsgSize(defaults.DefaultMaxSendMsgSize)),
 		}
 		if len(copts.dialOptions) > 0 {
 			gopts = copts.dialOptions
 		}
+		if len(copts.callOptions) > 0 {
+			gopts = append(gopts, grpc.WithDefaultCallOptions(copts.callOptions...))
+		} else {
+			gopts = append(gopts, grpc.WithDefaultCallOptions(
+				grpc.MaxCallRecvMsgSize(defaults.DefaultMaxRecvMsgSize),
+				grpc.MaxCallSendMsgSize(defaults.DefaultMaxSendMsgSize)))
+		}
```

Comment:
What happens if call options are passed, but `MaxCallRecvMsgSize` and `MaxCallSendMsgSize` are not among them?

Reply:
By default grpc use 4MB as limit https://docs.microsoft.com/en-us/aspnet/core/grpc/security?view=aspnetcore-6.0#message-size-limits. 
Do u think we should check if passed call options contains `MaxCallRecvMsgSize` or `MaxCallSendMsgSize`, if not use ours default setting?

Reply:
You could just order this differently such that our defaults first go into a `[]grpc.CallOption` then append the user options. The case where it is specified twice, the users will overwrite the default options

Reply:
@dmcgowan's solution sounds ideal.  It may be the case, at some point, that `grpc` defaults won't necessarily be suitable defaults for `containerd`.

Reply:
Thanks for suggestions!  Done